frontend/src/components/ViewTrips/trip-utils.test.js

```diff
@@ -16,56 +29,73 @@ describe('getCleanedTextInput tests', () => {
     const testRawName = 'Trip to No Man\'s Land';
     const expectedTripName = testRawName;
 
-    const testTripName = getCleanedTextInput(testRawName, testDefaultValue);
+    const testTripName = TripUtils.getCleanedTextInput(testRawName,
+                                                       testDefaultValue);
 
     expect(testTripName).toEqual(expectedTripName);
   });
 });
 
-const mockCurUserEmail = 'cur.user@email.com';
-jest.mock('../AuthUtils', () => ({
-    getCurUserEmail: () => mockCurUserEmail,
-    getUserUidArrFromUserEmailArr: (userEmailArr) => {
-          return new Promise(resolve => {
-            resolve(userEmailArr.map(userEmail => '_' + userEmail + '_'));
-          });
-        },
-}));
 describe('getCollaboratorUidArray tests', () => {
   test('No collaborators entered', async () => {
     const expectedUidArr = [`_${mockCurUserEmail}_`];
     // This is the list that is created when there are no collaborators added
     // (automatically one empty string from the constructor created ref).
     const testEmailArr = [''];
 
-    const testUidArr = await getCollaboratorUidArray(testEmailArr);
+    const testUidArr = await TripUtils.getCollaboratorUidArray(testEmailArr);
 
     expect(testUidArr).toEqual(expectedUidArr);
   });
 
   test('Some added collaborators', async () => {
-    const person1Email = 'p1@gmail.com';
-    const person2Email = 'p2@outlook.com';
     const expectedUidArr = [`_${mockCurUserEmail}_`,
-                            `_${person1Email}_`,
-                            `_${person2Email}_`];
-    const testEmailArr = [person1Email, person2Email];
+                            `_${USER_A_EMAIL}_`,
+                            `_${USER_Z_EMAIL}_`];
+    const testEmailArr = [USER_A_EMAIL, USER_Z_EMAIL];
```

Comment:
Is this outdated because you're not using the `__underscore__` method anymore? Correct me if I completely misunderstood what this test is for

Reply:
No I am still using the underscore method for mocking the AuthUtils function `getUserUidArrFromUserEmailArr`. I just replaced the `person1` and `person2` strings with the constants defined at the top for consistency sake.